tests/numerics/smoothness_estimator_03.cc

```diff
@@ -0,0 +1,72 @@
+#include <deal.II/dofs/dof_handler.h>
+
+#include <deal.II/fe/fe_nedelec.h>
+#include <deal.II/fe/fe_series.h>
+
+#include <deal.II/grid/grid_generator.h>
+#include <deal.II/grid/tria.h>
+
+#include <deal.II/hp/fe_collection.h>
+
+#include <deal.II/lac/vector.h>
+
+#include <deal.II/numerics/smoothness_estimator.h>
+
+#include "../tests.h"
+
+// This test is used to make sure FESeries::Fourier/Legendre
+// woring with non-primitive elements
```

Comment:
Would you move your brief test description before the `include` statements, and add the deal.II copyright preamble in the front of this file?

You could draw inspiration from the other smoothness_estimator tests.

```suggestion
// This test is used to make sure that FESeries::Fourier/Legendre
// work with non-primitive elements
```